bucket/peazip.json

```
@@ -0,0 +1,59 @@
+{
+    "version": "6.8.1",
+    "description": "A free file archiver utility, based on Open Source technologies",
+    "homepage": "https://www.peazip.org/peazip-portable.html",
+    "license": {
```

It is valid SPDX identifier

```suggestion
    "license": "LGPL-3.0-only",
```

---

I thought, where available, it's considered best practice to include both the identifier AND the app's specific page where it details the license.

---

Where did you get this information?

---

I saw it in a pull request or commit somewhere - I don't have time to track it down.

I think it makes sense to maximize application-specific information in the manifests, but I don't really care a huge deal.